cmd/openshift-install/targets.go

```diff
@@ -16,6 +16,7 @@ import (
 	"github.com/openshift/installer/pkg/asset/kubeconfig"
 	"github.com/openshift/installer/pkg/asset/manifests"
 	"github.com/openshift/installer/pkg/asset/metadata"
+	"github.com/openshift/installer/pkg/asset/util"
```

Comment:
i don't see this package addition in this commit.
Also please don't add util packages.

Reply:
It can be called something else, but we need a helper/util function outside of asset package to be able to unmarshal strings to actual type.